packages/react-components/react-migration-v8-v9/src/components/Checkbox/shimCheckboxProps.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { ICheckboxProps } from '@fluentui/react';
+import { CheckboxOnChangeData, CheckboxProps } from '@fluentui/react-components';
+import { getHTMLAttributes } from '../utils';
+
+// https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-components-checkbox-migration--page
```

Comment:
can we document what's the purpose of these links ?

Reply:
> can we document what's the purpose of these links ?

Yes, this link are the props of checkbox v8/v9 and the props mapping in fluent ui storybook. Have added these info in the comments. Thanks for your suggestion.